packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -264,6 +265,9 @@ class InkResponse extends StatefulWidget {
   /// material.
   final ValueChanged<bool> onHover;
 
+  /// The cursor for a mouse pointer when it enters or is hovering the region.
```

Comment:
what happens when I press a mouse button and it's no longer hovering?

Reply:
Mouse cursors are changed based on Enter and Exit (instead of Hover), therefore it does not rely on whether the pointer is down. The short answer is nothing will be affected.

If you want to change the cursor when the mouse is down, you can assign different cursors to `MouseRegion.cursor` depending on the state.